source/github-helpers/index.ts

```diff
@@ -21,9 +21,15 @@ Tested on isRepoTree, isBlame, isSingleFile, isEditFile, isSingleCommit, isCommi
 Example tag content on public repositories: https://github.com/sindresorhus/refined-github/commits/branch-or-commit/even/with/slashes.atom
 Example tag content on private repositories https://github.com/private/private/commits/master.atom?token=AEAXKWNRHXA2XJ2ZWCMGUUN44LM62
 */
-export const getCurrentBranch = (): string => {
+export const getCurrentBranch = (): string | undefined => {
 	// .last needed for #2799
-	return new URL(select.last<HTMLLinkElement>('[type="application/atom+xml"]')!.href)
+	const atom = select.last<HTMLLinkElement>('[type="application/atom+xml"]');
```

Comment:
Could be more descriptive as

```suggestion
	const feedLink = select.last<HTMLLinkElement>('[type="application/atom+xml"]');
```

Or `repoFeedLink` to be a bit more verbose?